README.md

```diff
@@ -1 +1,163 @@
-# blog-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog App <a name="about-project"></a>
+
+**Blog App** is an app created using Ruby on Rails. This app was inspired by Microverse for the Ruby on Rails practice.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Add Books, movies, games and music albums**
+- **Show the list of books, movies, games and music albums**
+- **Archive any item that is old**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Ruby installed on your computer
+
+### Setup
+
+Clone this repository to your desired folder or download the ZIP
+
+````
+
+-
+
+### Install
+
+Install this project with:
+
+No installation required
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  Open terminal in the folder directory and run "ruby main.rb"
+````
+
```

Comment:
- [Optional] Please kindly put the correct command, in this project there is no `main.rb` file so this command can't run the project. The correct command is `rails s` to run the app and you should add a command that people need to run before opening the project in the browser, the command help to install bundle for ruby `bundle install` 😁

Reply:
Done :hugs: